### ISSUE TITLE
pGMM

### ISSUE THREAD

**Issue by 02agarwalt** [Upward-Spiral-Science/fngs#112]
write pGMM algorithm, do methods_paper.md on it, and write proof of GMM. pGMM = prior-based GMM. w prior GMM, we know what the w matrix is ahead of time, and the pi-vector. Future iterations will constrain the expectation w vector to not vary from the prior we pass in.

**Comment by jovo** [comment]
@ebridge2 i'm not quite sure what this is.  i'd rather you do a "benchmarks", 
making a single page in "https://github.com/neurodata/checklists/blob/master/methods_paper.md" style, but including the methods that you have already (especially the synthetic data), 
so we have some things to compare against.  what do you think?

**Comment by ebridge2** [comment]
I agree; I just wanted to start somewhere more foundational for the tGMM before getting back there. As of now, the tGMM still does not converge in many cases, and I think it is a flaw in the algorithm itself, so what I was going to do was go to a much simpler model first to modify (we went from regular GMM -> tGMM; I think doing a pGMM before finishing tGMM will help us understand how to "better" do the tGMM in the next iteration, because I think that algorithm is one step ahead of us from a conceptual understanding right now). My goal for Magnetrons already is to finish that synthetic trial to be replicated on all 3; maybe a tbd by 2 weeks could be to have all 3 (wwCon, Null model, and pGMM) combined into one cohesive report, and then have a model due for tGMM++ as well, and then write up a methods_algorithms.md the following week (ie, 3 weeks out)?
